lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -1579,6 +1580,11 @@ EmitInfo WasmBytecodeGenerator::EmitM128BitSelect()
     EmitInfo mask = PopEvalStack(WasmTypes::M128);
     EmitInfo arg2Info = PopEvalStack(WasmTypes::M128);
     EmitInfo arg1Info = PopEvalStack(WasmTypes::M128);
+
+    ReleaseLocation(&mask);
+    ReleaseLocation(&arg2Info);
```

Comment:
temp register locations function as a stack, so last acquired should be first released (i.e. order here should be reversed)

Reply:
I've tried that, but it triggers `tmpReg == this->mNextLocation - 1` error. Looks like the arguments get requested in the reverse order (and that seems to be the case with other methods in this file as well) -- here they are taken off the eval stack, rather than requested.